build.gradle

```diff
@@ -64,14 +64,16 @@ sharedLibrary {
     testHarnessVersion = '1934.v90a_c07cf5b_21' // https://mvnrepository.com/artifact/org.jenkins-ci.main/jenkins-test-harness?repo=jenkins-releases
     pluginDependencies {
         // see https://mvnrepository.com/artifact/org.jenkins-ci.plugins/<name>?repo=jenkins-releases for latest
+        workflowCpsGlobalLibraryPluginVersion = '609.vd95673f149b_b'
```

Comment:
I think @gaiksaya you removed this library in the previous PR. Not sure why we want to add it back?https://github.com/opensearch-project/opensearch-build/pull/2899

Reply:
Looks like this is added to override the transient dependency version.

Reply:
@zelinh @gaiksaya Do you recommend explicitly updating the version of workflow-cps-global-lib . It is set to 2.14 now and remaining the same even for the latest versions of pipeline-groovy-lib and core Version. So far I found this solution to address the CVE's.